app/views/v2/common_cases_view.scala.html

```diff
@@ -0,0 +1,95 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.request.AuthenticatedRequest
+@import config.AppConfig
+@import views.html.includes.main
+@import views.html.components.heading
+@import views.html.components.common_atar
+@import views.html.components.common_liabilities
+@import views.html.components.primary_navigation
+@import views.html.components.common_correspondence
+@import views.html.components.common_miscellaneous
+
+@this(my_cases_secondary_nav: views.html.components.my_cases_secondary_navigation)
+
+@(title: String)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None, customTitle = Some(messages(title))) {
+
+    @primary_navigation()
+    @my_cases_secondary_nav()
+
+    <!--the heading component-->
+    @heading("heading text")
+
+    <div id="common-cases-tabs" class="govuk-tabs" data-module="govuk-tabs">
+        <ul class="govuk-tabs__list">
```

Comment:
Is there a way to make this tabbed-view a more generic component so that it can  be re-used with different lists of tabs in all of the dashboard views?

It's likely that the list of case types won't change, but in the future queues will be able to be added/removed/renamed so this will need to be dynamically driven somehow.

Reply:
As discussed with @periklas we can look at iterative improvements/generalisations as we do the subsequent tickets!